prod-maven-plugin/src/main/java/org/l2x6/cq/maven/prod/ProdInitMojo.java

```diff
@@ -226,6 +228,23 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     getLog().info("Setting camel-quarkus.extension.finder.strict = false in pom.xml");
                     props.addOrSetChildTextElement("camel-quarkus.extension.finder.strict", "false");
 
+                    /* Fix known problematic version properties in @sync comments */
+                    Optional<ContainerElement> jxmppVersion = props.getChildContainerElement("jxmpp.version");
+                    jxmppVersion.ifPresent(versionProperty -> {
+                        Comment comment = versionProperty.nextSiblingCommentNode();
+                        if (comment != null) {
+                            comment.setData(comment.getData().replace("camel.version", "camel-community-version"));
+                        }
+                    });
+
+                    Optional<ContainerElement> graalVMDocsVersion = props.getChildContainerElement("graalvm-docs.version");
+                    graalVMDocsVersion.ifPresent(versionProperty -> {
+                        Comment comment = versionProperty.nextSiblingCommentNode();
+                        if (comment != null) {
+                            comment.setData(comment.getData().replace("quarkus.version", "quarkus-community.version"));
+                        }
+                    });
+
```

Comment:
for `quarkus-community.version`, there are still `caffeine.version` and `sshd.version`. Can you check it?

Reply:
Well spotted!

You're right, I think we need to ensure they stay aligned to community too, so I'll add similar code for those cases.

Reply:
Nice work!